README.md

```diff
@@ -98,8 +125,7 @@ app.use(bodyParser.urlencoded({extended: true}))
 ```
 
 ```text
-Your answer...
-```
+The first line configures the app to use body-parser.  I think it should use const instead of var just to stay consistent with the material from the lessons.  The second line handles the json put request, and the third handles the form submissions.  The "body" in body parser is similar to that of params, but params is used exclusive for URL parameters in express.
 
```

Comment:
The difference between `var` & `let`/`const` is ES5 & ES6. If you go on to develop in javascript, it is probably inevitable that you will see both.